src/app/containers/ComscoreAnalytics/Amp/index.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const AmpComscoreAnalytics = () => (
+  <amp-analytics type="comscore">
```

Comment:
```suggestion
  <amp-analytics type="comscore" data-block-on-consent>
```

Reply:
I think we need this, otherwise this data will always be sent no matter whether the user has accepted the amp consent banner. This was missed off the original issue!

Reply:
Hmm this seems to block the request even if they have accepted. Though if they haven't accepted it does this request

https://sb.scorecardresearch.com/p?c1=2&c2=17986528&cs_ucfr=0&cs_amp_consent=insufficient&cs_pv=5111&c12=amp-qefSx4xAQXRJeNYaWL0ZHw&rn=0.7082436357631829&c8=Accueil%20-%20BBC%20News%20Afrique&c7=http%3A%2F%2Flocalhost%3A7080%2Fafrique&c9=&cs_c7amp=http%3A%2F%2Flocalhost%3A7080%2Fafrique.amp&comscorekw=amp

and if they have it does this request

https://sb.scorecardresearch.com/p?c1=2&c2=17986528&cs_ucfr=1&cs_amp_consent=sufficient&cs_pv=1817&c12=amp-qefSx4xAQXRJeNYaWL0ZHw&rn=0.5906800033587978&c8=Accueil%20-%20BBC%20News%20Afrique&c7=http%3A%2F%2Flocalhost%3A7080%2Fafrique&c9=&cs_c7amp=http%3A%2F%2Flocalhost%3A7080%2Fafrique.amp&comscorekw=amp

The main change being `cs_amp_consent=sufficient` rather than `cs_amp_consent=insufficient`.